controllers/ovndbcluster_controller.go

```diff
@@ -387,6 +387,10 @@ func (r *OVNDBClusterReconciler) generateServiceConfigMaps(
 	templateParameters := make(map[string]interface{})
 
 	templateParameters["OVN_LOG_LEVEL"] = instance.Spec.LogLevel
+	templateParameters["OVN_ELECTION_TIMER"] = instance.Spec.ElectionTimer
+	templateParameters["OVN_PROBE_INTERVAL_TO_ACTIVE"] = instance.Spec.ProbeIntervalToActive
+	templateParameters["OVN_ELECTION_TIMER"] = instance.Spec.ElectionTimer
+	templateParameters["OVN_PROBE_INTERVAL_TO_ACTIVE"] = instance.Spec.ProbeIntervalToActive
```

Comment:
Is there meant to be some sort of differentiation here?  The keys are duplicated, i.e. `OVN_ELECTION_TIMER` and `OVN_PROBE_INTERVAL_TO_ACTIVE` are used twice.  Should they be given some unique prefix or suffix, like `NB` or `SB` (and then adjust the `ovn-dbcluster-<type>.json` template files to use those unique keys)?

Reply:
seems just a leftover with the last update, so need cleanup.

Reply:
yeah, leftover. Removed :)